packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -92,6 +92,7 @@ export interface ITestContainerConfig {
 // new interface to help inject custom loggers to tests
 export interface ITestLoaderOptions extends ILoaderOptions {
```

Comment:
we should deprecate this type, and changer to ITestLoaderOptions = ILoaderOptions but use the ILoaderOptions from container-loader. then below, take ILoaderOptions  directly

Reply:
then if we add things we'll get them for free, rather than having to update this stuff

Reply:
We get them for free even with this since its an extension of ILoaderOptions right. configProvider is not part of ILoaderOptions but ILoaderProps. So maybe we should add that too in the create / load APIs and then we don't need to keep updating this.

Reply:
uggh. this isn't quite right. ideally the below just takes a partial `ILoaderProps` then we fill in the gaps. probably a bigger change than you want to make

Reply:
yeah that works too since ILoaderOptions is in ILoaderProps. Let me see how big the change will be.

Reply:
It was surprisingly simpler to replace it with ILoaderProps. I think it was only used to pass logger so it just worked.